examples/src/test/kotlin/ChangeStreamsTest.kt

```diff
@@ -136,6 +137,37 @@ internal class ChangeStreamsTest {
 
     }
 
+    // This test is getting ignored because the feature is only enabled for deployments on MongoDB 7.0
+    @Ignore
+    fun splitLargeChangeStreamTest() = runBlocking {
+        val changeEvents = mutableListOf<ChangeStreamDocument<Document>>()
+        // :snippet-start: split-large-change-stream
+        val pipeline = listOf(BsonDocument.parse("{ \$changeStreamSplitLargeEvent: {} }"))
```

Comment:
Question:
For personal edification, is this way way is more idiomatic than the following:
```
val pipeline = listOf(BsonDocument().append("$changeStreamSplitLargeEvent", BsonDocument()))
```